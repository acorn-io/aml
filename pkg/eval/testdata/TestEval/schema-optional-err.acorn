Schema: schema {
    foo?: 1
    bar: "hi"
}

{
    Schema
    {
        foo: "str"
        bar: "bye"
    }
}
