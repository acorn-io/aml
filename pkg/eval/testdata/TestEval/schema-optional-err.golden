"schema violation Schema.foo: expected kind number but got string with value (str) (schema-optional-err.acorn:1:16)"
